docs/scenarios_authoring/AggregatesInTimeWindows.md

```diff
@@ -19,9 +19,9 @@ Nussknacker implements 3 types of time windows - tumbling, sliding and session w
 
-A sliding-window in its default configuration (`emitWhenEventLeft` is set to `false`) and a tumbling-window with `On each event` emit mode enrich the incoming event with the value of the aggregate. If there were no aggregates "upstream" which terminated the events which entered scenario, the  events which entered the scenario will still be available downstream. As the variables are "attached" to the events, the `#input` and `#inputMeta` variables will be also available 'downstream'. 
+A sliding-window in its default configuration (`emitWhenEventLeft` is set to `false`) and a window with `On each event` emit mode (either tumbling-window or session-window) enrich the incoming event with the value of the aggregate. If there were no aggregates "upstream" which terminated the events which entered scenario, the  events which entered the scenario will still be available downstream. As the variables are "attached" to the events, the `#input` and `#inputMeta` variables will be also available 'downstream'. 
 
-Other window types - tumbling window in `After window closes` emit mode, and session window nodes behave differently - they generate a new event containing an aggregate when the time window is closed. Its timestamp is equal to the time of the timer that generated it, not system time of the moment when it happened. In other words the timestamp of the newly generated event which contains the aggregate will continue to use the notion of time used by events which the aggregate window saw. These nodes 'terminate' the events which entered them; consequently all the variables defined before the node (including `#input` and `#inputMeta`) will not be available downstream. The sliding-window behaves in the same way in its **non-default** configuration when parameter `emitWhenEventLeft` is set to `true` - it terminates events entering the aggregation node, emits the aggregate event when window is closed. 
+Other window types - tumbling window in `After window closes` emit mode, and session window in `After session end` nodes behave differently - they generate a new event containing an aggregate when the time window is closed. Its timestamp is equal to the time of the timer that generated it, not system time of the moment when it happened. In other words the timestamp of the newly generated event which contains the aggregate will continue to use the notion of time used by events which the aggregate window saw. These nodes 'terminate' the events which entered them; consequently all the variables defined before the node (including `#input` and `#inputMeta`) will not be available downstream. The sliding-window behaves in the same way in its **non-default** configuration when parameter `emitWhenEventLeft` is set to `true` - it terminates events entering the aggregation node, emits the aggregate event when window is closed.
```

Comment:
in `After session end` nodes -> in `After session end` mode

Reply:
changed